script.js

```diff
@@ -0,0 +1,59 @@
+// function to open a menu
+const openMenu= ()=>{
+    //menu.classList.toggle('show')
+    menu.style.display = "block"
+}
+const startIcon = document.querySelector(".bottom--start--icon");
+const menu = document.querySelector(".menu");
+const show= document.querySelector('.show')
+const hide = document.querySelector('.hide')
+
+// calling openMenu function to open menu 
+startIcon.addEventListener('click', openMenu)
+
+
+// to close menu if we click
+
+const closeMenu = document.querySelector('#menuCloseSpan')
+closeMenu.addEventListener('click', ()=>{
+    if(menu.style.display="block")
+    {
+        menu.style.display="none"
+    }
+})
+
+const showTime = document.querySelector(".showTime")
+
+// Defining Api function..
+ const myApi = async()=>{
+    try{
+        const res = await fetch(`https://timeapi.io/api/time/current/zone?timeZone=Australia%2FSydney`);
+        if (!res.ok) {
+            throw new Error("Cannot get data");
+        }
+        const data = await res.json();
+        const finalTime = new Date(data.dateTime).toLocaleTimeString(); // Convert to a readable time format
+        showTime.textContent = finalTime; // Set the time in the HTML
+    }
+    catch(error){
+        console.error("Error fetching time:", error);
+        showTime.textContent = "Error fetching time";
+    }
+}
+//Calling API for displaying Time
+myApi()
```

Comment:
You are only calling the time function once, which means if I have the app open for 10 mins I will see the time from 10 mins ago, call your function inside setInterval to update every second
```
setInterval(myApi, 1000)
```